ami-build/azp-minimal-x64.pkr.hcl

```diff
@@ -17,7 +17,7 @@ source "amazon-ebs" "envoy-azp-minimal-x64" {
     }
     most_recent = true
     # TODO: figure out why setting this (incorrectly) worked and uncomment
-    # owners = ["679593333241"]
+    owners = ["679593333241"]
```

Comment:
@mum4k i seem to recall you saying this is not the correct owner

when i tried to build the AMI without this it failed with an error about owner being required for sec reasons

uncommenting seems to have worked

Reply:
ill separate this change out as its not related to the fix here